api/api/controllers/cluster_controller.py

```diff
@@ -157,53 +157,6 @@ async def get_healthcheck(pretty: bool = False, wait_for_complete: bool = False,
     return json_response(data, pretty=pretty)
 
 
-async def get_nodes_ruleset_sync_status(pretty: bool = False, wait_for_complete: bool = False,
```

Comment:
I came up with this endpoint and figured it should be removed, so I did it in this PR to have everything cluster-related ready after it.